third_party/tonic/dart_wrappable.h

```diff
@@ -103,22 +104,32 @@ struct DartConverter<
     typename std::enable_if<
         std::is_convertible<T*, const DartWrappable*>::value>::type> {
   static Dart_Handle ToDart(DartWrappable* val) {
-    if (!val)
+    if (!val) {
       return Dart_Null();
-    if (Dart_WeakPersistentHandle wrapper = val->dart_wrapper())
-      return Dart_HandleFromWeakPersistent(wrapper);
+    }
+    if (Dart_WeakPersistentHandle wrapper = val->dart_wrapper()) {
+      auto strong_handle = Dart_HandleFromWeakPersistent(wrapper);
+      if (!Dart_IsNull(strong_handle)) {
```

Comment:
This seems odd. Why would we return null handle previously when attempting to realize a weak handle, but not anymore?  I daresay this would benefit from a comment.

Reply:
The new behavior or WPHs is that they point to Dart_Null() instead of auto-deleting themselves. I've added comments to clarify.